sdk/Trace/Tracer.php

```diff
@@ -232,6 +232,11 @@ public function getResource(): ResourceInfo
         return clone $this->resource;
     }
 
+    public function getTracerProvider(): TracerProvider
```

Comment:
Can this method be `private`?  It would be good to not add to the public API for SDK-internal purposes.

Reply:
Actually the reason I added it was because I needed to use it within `spanOptions` and I didn't want to make the private variable public. For me to get the correct Id generator, I need to pull it from the tracerProvider